tests/integration/_conftest/tokenizer.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any
+
+
+def tokenize(positional_args: list[Any], named_args: dict[str, Any]):
```

Comment:
Why not `args` & `kwargs`?

Reply:
`ProtostarArgument` names use dashes, hence they can't be represented as kwargs in Python.

Reply:
You can use dict:

```
def f(*args, **kwargs):
  print('args', type(args))
  for a in args:
    print(type(a), '/', a)
  print('kwargs', type(kwargs))
  for k, v in kwargs.items():
    print(type(k), type(v), '', k, '=>', v)

f(1, 2, 3, **{'a-a': 4, 'b': 5})
```

Reply:
If somebody starts using kwargs in this way:
```
f(foo=1, bar=2, baz=3, ...)
```
And later he needs to add kwarg with dash in its name, he needs to rewrite all arguments. Hence, it's better to require dict.

This shouldn't be a problem with args.

Reply:
Fair point